entrypoint.sh

```diff
@@ -20,20 +19,20 @@
 
 
 if [ "$1" = "test" ]; then
-    # Install standard test dependencies; YMMV
-    pip --quiet install \
-        .[test] nose PyHamcrest coverage
-    exec nosetests ${NAME}
+   # Install standard test dependencies; YMMV
+   pip --quiet install \
+       .[test] nose PyHamcrest coverage
+   exec nosetests
```

Comment:
I guess we'll want to move this to pytest at some point too, but out of scope here!

Reply:
Good thinking - I'll raise a ticket and add it to our backlog

Reply:
Have raised GLOB-54120 for this 👍